### PR TITLE
Census Household: Who Lives Here & Relationships

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
This PR introduced the `Who Lives Here` section including the relationships.
There is a lot of duplication which we can look at refactoring using jsonnet at a later stage.

There is other work that is needed to complete WLH:
There are numerous things that were missed in this PR which will be addressed at a later date.
1. Routing outlined in the card is not done as we currently cannot route based on answers in another section.
2. The Census Date placeholder on the interstitial page content on the `list` is currently hardcoded since validator doesn't support it and it not as simple as change the type in validator since it is an array.
3. On the list collector summary, the visitors' section title is missing the display address. Need to fix bug with the summary not having access to metadata.
4. List collector summary also displays other questions the user has answered. This will be resolved by further work by making use of the `hide_on_section_summary` key.
5. Relationship piping will need to be reworked once https://github.com/ONSdigital/eq-survey-runner/pull/2302 is in.
6. Relationships currently don't support descriptions in answer options.
7. Hub returns to the relationship question when clicking 'View answer'. This is the behaviour of the current implementation but the design want the user to go to the List Collector summary instead.

Currently, the validator will fail this on duplicate id, but I will raise a PR to fix this soon as it should be allowed when using variants.

### How to review 
Ensure:
1. The journey is the same as the prototype attached to the Trello card.

[Quick Launch for GB-ENG](https://v3-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-ENG&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-3193-build-who-lives-here-eng-wls-ni/data/en/census_household_gb_eng.json)

### CODE REVIEW

**Comment by MebinAbraham** [comment]
[Who Lives Here - To Do:](https://trello.com/c/2rCqLhri/3234-who-lives-here-to-do)

File: data/en/census_household_gb_eng.json [hunk 1]
```diff
@@ -0,0 +1,2069 @@
+{
```
Comment (MebinAbraham) [inline data/en/census_household_gb_eng.json]: Only commited for the purpose of Quick Launch. It will be removed once approved.

**Review by donnap573 — APPROVED** [review]

**Comment by MebinAbraham** [comment]
[Build Who lives here - Eng/Wls/NI (M)](https://trello.com/c/8nKwEebs/3193-build-who-lives-here-eng-wls-ni-m)

**Review by ajmaddaford — APPROVED** [review]
Nice one! 🎉

**Comment by JamesGardiner** [comment]
With regards to #2302, the schema will need to be reworked *before* that pr goes in.

**Pull request merged**